src/cli/src/lib.rs

```diff
@@ -15,6 +15,16 @@ mod profile;
 mod version;
 mod metadata;
 
+#[cfg(feature = "consumer")]
```

Comment:
would prefer to use cfg_if to make it tidy

Reply:
Moving this stuff into a module cleaned it up